### PR TITLE
OWDatasets: fix row height

### PR DESCRIPTION
**Pull request by markotoplak**
##### Issue
Row height was too small

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/widgets/data/owdatasets.py [hunk 1]
```diff
@@ -283,6 +283,7 @@ def info(file_path):
             item1 = QStandardItem()
             item1.setData(" " if datainfo.islocal else "", Qt.DisplayRole)
             item1.setData(datainfo, Qt.UserRole)
+            item1.setData(QSize(26, 26), Qt.SizeHintRole)
```
Comment (lanzagar) [inline Orange/widgets/data/owdatasets.py]: On my system this looks quite big.
24 still results in higher rows but looks better IMO.

**Comment by markotoplak** [comment]
Why do we need this at all?

**Comment by kernc** [comment]
This is how it looks here. The lines with target pixmaps are higher than the lines without.
screenshot_2017-11-13_13-18-35
While I'd never wish to advocate for this, some padding is in.

**Comment by ales-erjavec** [comment]
@kernc Can you please check gh-2774

**Comment by janezd** [comment]
Closed in favour of #2774.

**Pull request closed without merging**